docs/1.-Welcome/Documentation-Guidelines/Other Documentation Types/Readme-files-in-code-repositories.md

```diff
@@ -41,22 +43,91 @@ Provide a brief overview of the repository, and the problem it aims to solve.
 
 List the major technologies and frameworks used within the repository, as a bullet-point list. Ideally link out to some documentation for each item, this is especially important for technologies and frameworks outside our usual tech stack.
 
+Bullet point list of any Quartex Microservices that are used within this repository.
+
+Bullet point list any storage used.
+
+### Architecture
```

Comment:
We might want to make this section applicable only to microservice README files. Thoughts, @rhodh?